content/integrations/java.md

```diff
@@ -336,8 +336,10 @@ instances:
   5. Restart the agent: `sudo /etc/init.d/datadog-agent`
 
 
+## Further Reading
 
-
-
-
-
+* [I Have a Matching Bean for my JMX integration but nothing on Collect !](https://help.datadoghq.com/hc/en-us/articles/218277463-I-Have-a-Matching-Bean-for-my-JMX-integration-but-nothing-on-Collect-)
```

Comment:
Is the extraneous space between Collect and the ! intentional?

Reply:
it to have more impact :) reviewed 👍